ansibledriver/service/ansible.py

```diff
@@ -288,18 +292,33 @@ def v2_runner_on_start(self, host, task):
     def runner_on_failed(self, host, res, ignore_errors=False):
         logger.info('runner_on_failed {0} {1}'.format(host, res))
 
+    def __handle_unreachable(self, result):
+        # TODO do not overwrite if already set
+        self.failed_task = result._task.get_name()
```

Comment:
Do you need to do something for this TODO?

Reply:
I copied this block of code from the `v2_runner_on_unreachable` and it already included this TODO. Didn't remove it as I didn't know if it was still valid.